rafthttp/aggregate_logger.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2015 CoreOS, Inc.
```

Comment:
Why do we put in the rafthttp pkg at the first place? This should not be here. :(